### PR TITLE
Add "Previous Commits" row to experiments table

### PR DESCRIPTION
**Pull request by julieg18**
## Demo

https://user-images.githubusercontent.com/43496356/211624306-f1b47b3e-fdde-4515-8122-1e693b2d39e4.mov

Part of #3036

### CODE REVIEW

File: webview/src/experiments/components/table/styles.module.scss [hunk 1]
```diff
@@ -566,6 +566,19 @@ $bullet-size: calc(var(--design-unit) * 4px);
       outline-offset: -4px;
     }
   }
+
+  .otherCommitsRow {
```
Comment (julieg18) [inline webview/src/experiments/components/table/styles.module.scss:570]: Currently the row has no hover animation but we could change the background color on hover like we do with other rows if preferred.

Comment (shcheklein) [reply webview/src/experiments/components/table/styles.module.scss]: no need for this I think

File: extension/src/test/e2e/extension.test.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe('Experiments Table Webview', function () {
   const webview = new ExperimentsWebview('experiments')
 
   const epochs = 15
-  const headerRows = 3
+  const headerRows = 4
```
Comment (mattseddon) [inline extension/src/test/e2e/extension.test.ts:40]: [N] I would say it's an extra commit row, not a header (not at the top of the table)

Comment (mattseddon) [reply extension/src/test/e2e/extension.test.ts:40]: or specifically, call out "pastCommitsRow"

**Review by mattseddon — APPROVED** [review]

**Pull request merged**